### PR TITLE
Updates: Dims out fields that are being edited by someone else

### PR DESCRIPTION
**Pull request by jcontonio**
Grays them out and disables any mouse interaction

### CODE REVIEW

**Review by dheniges — CHANGES_REQUESTED** [review]
This disables all field types, including text fields. I don't think we want that?

**Comment by jcontonio** [comment]
Oh duh

**Comment by jcontonio** [comment]
Updated

File: web/src/client/js/components/DocumentFields/DocumentFieldComponent.js [hunk 1]
```diff
@@ -22,6 +22,12 @@ const DATA_FIELD_TYPES = [
   FIELD_TYPES.STRING,
 ]
 
+const FIELDS_THAT_DISABLE_WHEN_EDITING = [
```
Comment (dheniges) [inline web/src/client/js/components/DocumentFields/DocumentFieldComponent.js]: don't need this now?

**Review by dheniges — CHANGES_REQUESTED** [review]
Works good now, looks like just a leftover constant whitelist you started?

**Review by dheniges — APPROVED** [review]
:sealeel:

**Pull request merged**